src/sentry/integrations/vsts/integration.py

```diff
@@ -463,8 +468,36 @@ def build_integration(self, state: Mapping[str, Any]) -> Mapping[str, Any]:
             integration_model = IntegrationModel.objects.get(
                 provider="vsts", external_id=account["accountId"], status=ObjectStatus.ACTIVE
             )
-            # preserve previously created subscription information
-            integration["metadata"]["subscription"] = integration_model.metadata["subscription"]
+
+            # Get Integration Metadata
+            integration_migrated = integration_model.metadata.get("integration_migrated", False)
```

Comment:
Nit: It might be good to change this to a version number + check in case we need to conditionally migrate again.